pytorch_lightning/trainer/data_loading.py

```diff
@@ -237,3 +181,77 @@ def determine_data_use_amount(self, train_percent_check, val_percent_check,
             self.train_percent_check = overfit_pct
             self.val_percent_check = overfit_pct
             self.test_percent_check = overfit_pct
+
+    def _percent_range_check(self, name):
+        value = getattr(self, name)
+        msg = f"`{name}` must lie in the range [0.0, 1.0], but got {value:.3f}."
+        if name == "val_check_interval":
+            msg += " If you want to disable validation set `val_percent_check` to 0.0 instead."
+
+        if not 0. <= value <= 1.:
+            raise ValueError(msg)
+
+    def _ddp_sampler_check(self, data_loader, mode):
+        """
+        Check if sampler is used correctly in ddp mode
+        :param data_loader:
```

Comment:
add datatypes

Reply:
also make the docs consistent with our formatting:

Args:
   dfdf etc...

Reply:
The formatting is different across files. So, I am not sure which to follow

Reply:
we converged to napoleon :]